components/eam/docs/user-guide/index.md

```diff
@@ -1 +1,139 @@
+```
+
+#### Aerosol physical properties
+
+The aerosol properties files provide aerosol refractive index, density, and aerosol hygroscopicty information for each aerosol species, as well as information about lognormal mode definition and lookup tables of polynomial expression coefficients for aerosol optics calculation for each mode. These aerosol physical and chemical properties are used by the radiation, aerosol microphysics and other related source and sink processes, and droplet activation/ice nucleation schemes. Detailed information on the files and related references can be found [here](aerosol_phys_prop.md).
+
+#### Aerosol and gas emission and oxidant files 
+
+Details of the aerosol and gas emission and oxidant files used in various historical, present-day, and future scenario can be found [here](emission_oxidant_files.md).
+
+#### Linoz v3 input files
+
+Linozv3 uses the ozone tendency, (net production minus loss) calculated from its climatological mean state (function of month, latitude, altitude) and a first-order Taylor series expansion about the local ozone, temperature, and overhead ozone column. For historical simulation, Linozv3 uses the linozv3 data files with monthly resolution, spanning the dates 1849-01 -- 2014-12.
+
+##### Historical files
+
+```
+ linoz_data_file                = ‘linv3_1849-2101_CMIP6_Hist_10deg_58km_c20231207.nc’
+ linoz_data_path                = '/lcrc/group/e3sm/data/inputdata/atm/cam/chem/trop_mozart/ub'
+ linoz_data_type                = 'INTERP_MISSING_MONTHS'
+```
+
+Refer to [this page](https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/3764486280/Production+of+the+Linoz+v3+data) for more details on Linoz v3 input files. 
+
+#### Topography
+The global elevation on the atmosphere grid is a key input dataset.  The dataset is grid dependent. It contains the geopotential data (on both the GLL dynamics and PG2 physics grids) and two surface roughness quantities, `SGH` and `SGH30`.
+
+EAMv3 NE30 data:  
+```
+/lcrc/group/e3sm/data/inputdata/atm/cam/topo/USGS-gtopo30_ne30np4pg2_x6t-SGH.c20210614.nc'
+```
+
+This file is computed via a complex procedure that starts with high resolution dataset (for EAMv3, we use the GTOPO30, a 30 arc-second resolution data set on a lat-lon grid) that is then downsampled to a 3km cubed-sphere grid (cube3000) and then downsampled to the atmosphere  grid on the (GLL nodes), and then smoothed with the same viscosity operator used by the dycore. The smoothed GLL topography is then mapped to the PG2 grid. Finally, two different surface roughness fields are computed:
+
+- `SGH30`: the variance between GTOPO30 and GTOPO30-downsampled-to-PG2 (independent of any dycore specific smoothing).     (used by CLUBB, TMS, vertical_diffusion)
+- `SGH`:  the variance between the cube3000 data and the smoothed PG2 data.  (used by GWD parameterizations)
+
+#### Land-use / land cover change
+
+- <span style="color:red">Info needed on land-use land cover change / land surface data</span>
```

Comment:
@bishtgautam - Could I get some help with a short description of the land-use land-cover change that we would use in the F2010, F1850, and F20TR simulations?